src/client/components/app/LineDetailsDisplay/PositionsTable.tsx

```diff
@@ -187,7 +189,7 @@ export const PositionsTable = (props: PositionsProps) => {
     if (event.status === 'PROPOSED' && userRoleMetadata.role === BORROWER_POSITION_ROLE) {
```

Comment:
can we rename this function i have no idea what its doing or the data thats returned. @Seroxdesign